modules/tutorials/pages/pattern-matching/multiple-hop-and-accumulation.adoc

```diff
@@ -566,13 +553,13 @@ INTERPRET QUERY () SYNTAX v2 {
 }
 ----
 
-*Example 2.* Find Viktor Akhiezer's liked posts' authors A, and his liked comments' authors B. Count the common universities that both A and B have members studied at.
+*Example 2.* Find Viktor Akhiezer's liked posts' authors A, and his liked comments' authors B. Count the universities that members from groups A and B studied at.
```

Comment:
I think the rewrite loses the sense that we only want universities that were attended by students from both A and B.  The word "both" is important.